classes/itemlist.php

```diff
@@ -791,41 +791,58 @@ public function display_relations_table() {
     }
 
     /**
-     * Adds elements to an array starting from initial conditions.
-     *
-     * Conditions are array('hidden' => 0) OR array('reserved' => 0)
-     * Add to $nodelist all the children that will inherit the parent condition
+     * Get the recursive list of children of a specific item.
+     * This list counts children and children of children for as much generation as it is.
      *
-     * @param integer $nodelist
-     * @param array $additionalcondition
-     * @return void
+     * @param int $oldestparent
+     * @param array $where
+     * @return object $childrenitems
      */
-    public function add_child_node(&$nodelist, $additionalcondition) {
+    public function item_get_children($oldestparent=null, $where=null) {
```

Comment:
So, the function is expected to return the parent item and all the tree hanging from it. Surely I'd suggest to change the name of the parameter. `$oldestparent` doesn't look correct, why is it "old"? You can name it `rootitem`, `baseitem` or things like that... but "old" ?

Reply:
Also, surely proper phpdocs help to understand the params better.